app/assets/javascripts/miq_application.js

```diff
@@ -734,6 +734,23 @@ function miqChartMenuClick(itemId) {
   }
 }
 
+function miqRESTAjaxButton(url, button) {
+  var form = $(button).parents('form:first')[0];
+  if (form) {
+    $(form).submit(function(e) {
+      e.preventDefault();
+      return false;
+    });
+    miqJqueryRequest(form.action, {
+      beforeSend: true,
+      complete: true,
+      data: $(form).serialize()
```

Comment:
More info on the ```form serialize``` issue -
Some recent change in one of the hamls is probably causing this, since I distinctly remember that this was working earlier.
Currently investigating the haml changes...